tools/common.py

```diff
@@ -0,0 +1,127 @@
+# Common tools class
+"""Class for aioshelly cmdline tools."""
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Any, cast
+
+import aiohttp
+
+import aioshelly
+from aioshelly.block_device import BLOCK_VALUE_UNIT, COAP, BlockDevice, BlockUpdateType
+from aioshelly.common import ConnectionOptions
+from aioshelly.const import BLOCK_GENERATIONS, MODEL_NAMES, RPC_GENERATIONS
+from aioshelly.exceptions import InvalidAuthError, ShellyError
+from aioshelly.rpc_device import RpcDevice, RpcUpdateType, WsServer
+
+coap_context = COAP()
+ws_context = WsServer()
+
+
+class CommonTools:
+    """Common method for building tools."""
```

Comment:
I don't think this needs to be a class as its doesn't ever set anything on `self`.  It looks like be a simple library of functions.

Reply:
Done